docs/editors/vscode.md

```diff
@@ -1,13 +1,11 @@
 ---
 id: vscode
+sidebar_label: VS Code
 title: Visual Studio Code
 ---
 
 ![Goto Definition](https://user-images.githubusercontent.com/1408093/48776422-1f764f00-ecd0-11e8-96d1-170f2354d50e.gif)
 
-```scala mdoc:requirements
```

Comment:
Why is this removed? We should keep it, it's the https://scalameta.org/metals/docs/editors/vscode.html#requirements section.